### PR TITLE
Update FAQ, adding question related to trademark.md

### PR DESCRIPTION
**Pull request by irelativism**
Adding address to the trademark update.

And Answering such policy moving forward, related to such question

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -116,6 +117,20 @@ Still, there are a few things we can say now:
...
+<a id="q-trademark"></a>
+**🤔 Q: Will the Rust trademark lacunes, that derived from Mozilla media-guide be resolved?**
+
+💁 A: We’re still finalizing all the details around the foundation and by consequence in the trademark, nontheless we are working on a python, perl "like" policy, so our branding can still be efectively protected, to prevent copycat, or malicius forks, that intend to inject malware on our users systems.
+
+But when such work is completed, the following will be adressed:
+
+* Removing past restrictions on our contributors when they intend to give back or patch our project codebase.
+* Respect the freedom to improve the program, and release it back to the Rust coebase, in order to respect FOSS ideals, removing the clauses were demanding permissions are necessary.
+* Remove all mozzilla media-guide references, so such restrictive policy can be removed.
```
Comment (bjorn3) [inline FAQ.md:129]:
```suggestion
* Remove all mozilla media-guide references, so such restrictive policy can be removed.
```

**Comment by irelativism** [comment]
the issue #35 is related to this questioning such. Cheers in advance for fedback.

**Review by irelativism — COMMENTED** [review]
sounds good! Cheers

**Comment by nikomatsakis** [comment]
Hey -- as we wrote in https://github.com/rust-lang/foundation-faq-2020/issues/35, while we do expect the foundation to review the trademark policy, it will be up to the board to decide what terms are selected. In general, we did not intend for people to be opening PRs directly against the FAQ, so I’m going to go ahead and close this PR.

**Pull request closed without merging**